test/with-timers.test.js

```diff
@@ -0,0 +1,51 @@
+import t from 'tap'
+import Fastify from 'fastify'
+import FakeTimers from '@sinonjs/fake-timers'
+import slowDownPlugin from '../index.js'
+import { HEADERS } from '../lib/constants.js'
+
+t.test('Check the response time with 2 enpoints', async t => {
```

Comment:
this test is too hard to read. I'm sure there's a way to test the same thing in much simpler ways